packages/utils/src/useExport/useExport.js

```diff
@@ -0,0 +1,40 @@
+import { callCallback, filename, linkAndDownload, csvForItems, jsonForItems, exportableColumns } from './helpers';
+
+const useExport = ({ exporter, columns = [], isDisabled = false, onStart, onComplete, onError }) => {
+  const exportColumns = exportableColumns(columns);
+  const exportWithFormat = async (format) => {
+    callCallback(onStart);
+    const items = await exporter()
+      .then((items) => {
+        callCallback(onComplete, items);
```

Comment:
Safe accessor can be used instead of callCallback
```suggestion
        onComplete?.(items);
```